### ISSUE TITLE
Remove bluetooth.svg from Numix base, keep in Numix-Circle

### ISSUE THREAD

**Issue by dirtydancing** [numixproject/numix-icon-theme#438]
In the course of testing different inheritance possibilities, cf. https://github.com/numixproject/numix-icon-theme/issues/190#issuecomment-90890423, I came across an inconsistency with the Bluetooth icon: `bluetooth.svg` currently is a symlink in Numix base, pointing to the target `bluetooth-active.svg`. 

But the icon names for the Bluetooth status are `bluetooth-active.svg` and `bluetooth-disabled.svg` , so `bluetooth.svg` is not needed here.

To make things worse, `bluetooth.svg` is displayed in the system settings overview in Ubuntu. This should be a consistent Circle icon if Numix-Circle is selected (`bluetooth.svg` exists in Numix-Circle and should be kept there), and it should be the original icon in the system settings overview if Numix base is selected, not the `bluetooth-active.svg` status icon (target icon) from Numix base.

In conclusion, I would like to suggest to remove `bluetooth.svg` from Numix base.

@wa4557 If you fancy to take a look: Can you think of a reason to keep `bluetooth.svg` in Numix base?

**Comment by wa4557** [comment]
@dirtydancing No. In fact if we keep it and go along with the renaming and `apps` folders approach, this icon should be renamed, so if it is deleted or renamed doesn't make a lot of difference...

**Comment by dirtydancing** [comment]
@wa4557 Agreed. Actually, I checked this with your testing branch of numix-icon-theme, the one that already has the renamed icons in it (and the `apps` folders), and `bluetooth.svg` is still present there (not renamed yet).

Given the choice for this icon between renaming and deleting, I think I would go for deleting, as `blueetooth.svg` as an "app status icon" appears not to be needed in Numix base at all (also not "re-renamed").

The only reason for keeping it and renaming it rather than deleting might be that another distribution could be using this specific icon for the panel (?). So I guess renaming would be fine as well.

Here are screenshot with Numix base and Numix-Circle, both via PPA, to demonstrate what I mean with the system settings overview:

Numix-Circle (all fine)
![numix-circle](https://cloud.githubusercontent.com/assets/7164227/7045559/7ef416ee-ddff-11e4-9b81-9229b5f0b08b.png)

Numix base (current)
![numix](https://cloud.githubusercontent.com/assets/7164227/7045582/bbc80da0-ddff-11e4-8e7a-94df2c1a81cb.png)

Numix base (proposed without `bluetooth.svg`)
![numix_proposed](https://cloud.githubusercontent.com/assets/7164227/7045588/ccc16cc8-ddff-11e4-9057-c33d4dd64393.png)

**Comment by wa4557** [comment]
@dirtydancing what's going on with "Information" "User (Benutzer)" "Languages (Sprachen)" etc... Why are there the wrong /no icons displayed ?

**Comment by dirtydancing** [comment]
@wa4557 Haven't checked those. As this is Numix base, inheriting `gnome,hicolor`, normally those should at least be in the fallback icon themes. Can you confirm that they are currently not displayed at all in Ubuntu 14.10?

**Comment by wa4557** [comment]
Positive, not displayed in Ubuntu 14.10...

**Comment by dirtydancing** [comment]
Well, this is what the system settings look like with `gnome` and with `hicolor` shipping with Ubuntu 14.10:

![hicolor](https://cloud.githubusercontent.com/assets/7164227/7049116/68f88182-de17-11e4-856a-1c3c9953f9b5.png)

This means that there are no fallback icons for these (some are filled and Wacom is replaced by Numix icons). I figure as long as there is no issue report specifically addressing this, there is no use case and nothing to fix :-) OK, we might just have opened this box, but this would be a new issue report. Apparently, for a user using Numix base ootb on Ubuntu 14.10, currently there will be icons missing.

As a side note: `Humanity-Dark` icon theme (default Ubuntu icon theme `ubuntu-mono-dark` inherits from `Humanity-Dark,gnome,hicolor`) looks like this:

![humanity-dark](https://cloud.githubusercontent.com/assets/7164227/7049195/ff26d154-de17-11e4-9b9f-49dfb495edb4.png)

**Comment by dirtydancing** [comment]
Additional note: the same other issue still exists in Ubuntu 15.04 daily. Screenshots with PPA versions:

Numix-Circle (complete)
![1504daily_numix-circle](https://cloud.githubusercontent.com/assets/7164227/7049724/23932f0c-de1c-11e4-99ee-e3217b3a5f07.png)

Numix base (icons missing, `bluetooth.svg` issue not fixed yet)
![1504daily_numix](https://cloud.githubusercontent.com/assets/7164227/7049732/3ba5c80c-de1c-11e4-974d-8887cf6d682b.png)

**Comment by dirtydancing** [comment]
Separate issue report opened at https://github.com/numixproject/numix-icon-theme/issues/442.
